core/src/main/java/com/web3auth/core/Web3Auth.kt

```diff
@@ -171,7 +213,7 @@ class Web3Auth(web3AuthOptions: Web3AuthOptions) {
      */
     private fun authorizeSession(): CompletableFuture<Web3AuthResponse> {
         val sessionCompletableFuture: CompletableFuture<Web3AuthResponse> = CompletableFuture()
-        val sessionResponse: CompletableFuture<String> = sessionManager.authorizeSession(true)
+        val sessionResponse: CompletableFuture<String> = sessionManager.authorizeSession(false)
```

Comment:
why is this false now?

Reply:
Since encryption and ecryption happening in native android side.